src/CMakeLists.txt

```diff
@@ -2,17 +2,15 @@ configure_file(CabanaPD_config.hpp.cmakein CabanaPD_config.hpp @ONLY)
 
 file(GLOB HEADERS GLOB *.hpp)
 
-file(GLOB SOURCES *.cpp)
-
 install(FILES ${HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/CabanaPD_config.hpp DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
-add_library(CabanaPD ${SOURCES})
```

Comment:
@streeve : What does this change do?

Reply:
This makes it a header-only library (there are no source files anymore)